source/io/module_eggShell.js

```diff
@@ -679,6 +681,8 @@
         };
 
         Module.eggShell.setOccurrence = Occurrence_Set;
+
+        Module.eggShell.occurEvent = publicAPI.eggShell.occurEvent = OccurEvent;
```

Comment:
I think we should expose this from Module.controlEvents in the module_controlEvents.js file instead.

Reply:
I'm not sure; the OccurEvents entry point is generic and will also be used to fire events on VIs (mouse down, move, etc.) when they are supported, and could also be used to fire User Events from JS if someone wanted to do that.

Reply:
Then maybe the module should be renamed to something more generic, module_eventHelpers or something

Reply:
I'll leave this to Sid or Jared if they want to make this change, as they wrote the JS portions of this PR.

Reply:
Just so they are aware @siddhukrs @segaljared